metadata-io/src/main/java/com/linkedin/metadata/entity/EntityService.java

```diff
@@ -1070,11 +1070,6 @@ public List<Pair<String, RecordTemplate>> generateDefaultAspectsIfMissing(@Nonnu
       aspectsToGet.add(DATA_PLATFORM_INSTANCE);
     }
 
-    boolean shouldHaveStatusSet = isAspectMissing(entityType, STATUS, includedAspects);
```

Comment:
Just to make sure - we don't do this anywhere else right?

Reply:
We set status removed to true when soft deleting key aspects for entities. Nothing else.